modules/xmpp/SignalingLayerImpl.js

```diff
@@ -357,6 +354,25 @@ export default class SignalingLayerImpl extends SignalingLayer {
         return false;
     }
 
+    /**
+     * @inheritDoc
+     */
+    setTrackSourceName(ssrc, sourceName) {
+        if (typeof ssrc !== 'number') {
+            throw new TypeError(`SSRC(${ssrc}) must be a number`);
+        }
+
+        // Now signaling layer instance is shared between different JingleSessionPC instances, so although very unlikely
+        // an SSRC conflict could potentially occur. Log a message to make debugging easier.
+        const existingName = this._sourceNames.get(ssrc);
+
+        if (existingName && existingName !== sourceName) {
+            logger.error(`SSRC(${ssrc}) sourceName re-assigned from ${existingName} to ${sourceName}`);
```

Comment:
Shouldn't we add the check for ssrc rewriting support?

Reply:
Yep, it needs to be an error when ssrc-rewriting is not enabled, a debug message otherwise.